infra/pr_helper.py

```diff
@@ -88,7 +95,7 @@ def main():
       repo_url = new_project.get('main_repo')
       if repo_url is None:
         message += (f'{pr_author} is integrating a new project, '
-                    'but the `repo_url` is missing. '
+                    'but the `main_repo` is missing. '
```

Comment:
Will this modification overlap with [this one](https://github.com/google/oss-fuzz/pull/10553/files#diff-8cde736463fcc3019311b0661eed74d119d2103c6f1754dcbeeb50e8f7664733R91)?

Reply:
I am not sure if we want to make change to the doc file.